addons/l10n_co/i18n/l10n_co.pot

```diff
@@ -2864,7 +2959,7 @@ msgstr ""
 #: model:account.account,name:l10n_co.1_co_puc_240805
 #: model:account.account,name:l10n_co.4_co_puc_240805
 #: model:account.account.template,name:l10n_co.co_puc_240805
-msgid "VAT Payable"
+msgid "Sales VAT"
```

Comment:
not sure if this is ok but we need to update the name

Reply:
Hi @aliyatastemirova for this description, the term in spanis is "IVA Generado" we can switch it to "Accrued VAT", this translations makes more sence indeed. 

Regarding the account, the group 2408 is used for the VAT and the the subacounts 240810 is used for vendor bills (credited VAT)  and 240805 are used in the sales transactions (Debited VAT) this level of detail is not included in the PUC documentation but it's the common practice in CO, here are some references: 
https://i.imgur.com/B2YEzUH.png 
https://www.gerencie.com/la-contabilidad-debe-reflejar-las-operaciones-gravadas-excluidas-y-exentas-de-iva.html#:~:text=A%20manera%20de%20ejemplo%20podemos,diferencial%20que%20pueda%20existir%2C%20etc. 

cc. @vbe-odoo @fvz-odoo

Reply:
@ren-odoo thanks, updated the name :slightly_smiling_face: